environments/arcus/inventory/group_vars/all/bastion.yml

```diff
@@ -0,0 +1,3 @@
+bastion_user: slurm-app-ci
+bastion_ip: 128.232.222.183
```

Comment:
How come this is needed if the cluster can be accessible from the internet?

Reply:
There's no floating IP on any nodes (the same as  `smslabs`).

Reply:
Fair enough!